core/market/tests/test_cyclic_broadcasts.rs

```diff
@@ -62,7 +62,7 @@ async fn test_startup_offers_sharing() {
 
     // We expect, that after this time we, should get at least one broadcast
     // from each Node.
-    tokio::time::delay_for(Duration::from_millis(400)).await;
+    tokio::time::delay_for(Duration::from_millis(800)).await;
```

Comment:
We need to reduce the number of subscribes being processed, because it seems CI machines @ GH are not able to process such an amount of data within 100ms. (production code has broadcasts every 4mins which should suffice even for weak machines).